### ISSUE TITLE
Urticaria-like rash

### ISSUE THREAD

**Issue by jpazmandi** [obophenotype/human-phenotype-ontology#6119]
**Preferred term label:**
Urticaria-like rash

**Synonyms**
Hive-like rash

**Definition (free text, please give PubMed ID)**
Urticaria (hives) is a vascular reaction of the skin marked by the transient appearance of smooth, slightly elevated papules or plaques (wheals) that are erythematous and that are often attended by severe pruritus.

**Parent term (use hpo.jax.org/app)**
HP:0000988

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-3411-9598

**Comment by pnrobinson** [comment]
@jpazmandi  How is this different from simply Urticaria? https://hpo.jax.org/app/browse/term/HP:0001025

**Comment by jpazmandi** [comment]
urticaria has eosinphils on biopsy 
urticaria-like rash has neutrophils on biopsy - hence is not urticaria

**Comment by pnrobinson** [comment]
Do you mean this: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7256433/ ?

**Comment by jpazmandi** [comment]
Yes, thank you!

**Comment by pnrobinson** [comment]
added as HP_0033167
